source/documentation/administration_guide/topics/conc_maxfreememoryforoverutilized_and_minfreememoryforunderutilized_scheduling_policy_properties.adoc

```diff
@@ -36,6 +36,13 @@ If you define the *MaxFreeMemoryForOverUtilized* and *MinFreeMemoryForUnderUtili
 
 * If *MinFreeMemoryForUnderUtilized* is not defined, only overutilized hosts are source hosts, and hosts that are not overutilized are destination hosts.
 
+* To prevent the host from over utilization of all the physical CPUs, define the virtual CPU to physical CPU ratio - *VCputophysicalCpuRatio* with a value between 0.1 and 2.5. When this parameter is set, hosts with a lower CPU utilization are preferred when scheduling a virtual machine.
```

Comment:
```suggestion
* To prevent the host from over utilization of all the physical CPUs, define the virtual CPU to physical CPU ratio - *VCpuToPhysicalCpuRatio* with a value between 0.1 and 2.9. When this parameter is set, hosts with a lower CPU utilization are preferred when scheduling a virtual machine.
```